### PR TITLE
#3120

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/3120 - [studio-ui] Preview Assets in Browse page for s3, cmis webdav is not working #3120

### CODE REVIEW

**Review by rart — COMMENTED** [review]
@vestrada please confirm: 

The FTLs contain markup for what looks to be a pop up (modal dialog). Are these related to the recent hide/destroy issue?

Since these dialogs are on the FTL destroying them might cause issues reopening them without refreshing the page. 

Do all of the hide/destroy fixes work when being opened various times without refreshing the page?

Why are these pop ups on the ftl but are basically blank?

**Comment by vestrada** [comment]
No, It is not related. This is a custom pop up created for browse page, to preview a specific asset, I was just changing it because seems like there was a change when the search page was created. In this case there is just one dialog that will preview different assets. And it isn't yui dialog.

File: static-assets/components/cstudio-browse/browseWebDAV.js [hunk 1]
```diff
@@ -222,7 +222,7 @@
         $('.cstudio-wcm-result .results').delegate( ".magnify-icon", "click", function() {
             var path = $(this).attr('data-source');
             var type = $(this).attr('data-type');
-            me.magnify(path, type);
+            CStudioAuthoring.Utils.previewAssetDialog(path, type);
```
Comment (rart) [inline static-assets/components/cstudio-browse/browseWebDAV.js:225]: Will `CStudioAuthoring.Utils.previewAssetDialog` make use of the markup on the FTL to create the dialog? There's no FreeMarker output or unique differences between the different browse FTLs; it looks like duplicated HTML. 

Refactor for the pop up template to be on the JavaScript `previewAssetDialog` function and remove the HTML from the FTLs.

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by vestrada** [comment]
Done

**Review by rart — APPROVED** [review]

**Pull request merged**